src/sidebar/templates/annotation.html

```diff
@@ -149,37 +149,33 @@
 
     <div class="annotation-actions" ng-if="!vm.isSaving && !vm.editing() && vm.id()">
       <div ng-show="vm.isSaving">Saving…</div>
-      <button class="btn btn-clean annotation-action-btn"
+      <annotation-action-button
+        icon="'h-icon-annotation-edit'"
+        is-disabled="vm.isDeleted()"
+        label="'Edit'"
```

Comment:
The use of `aria-label` is an implementation detail as far as the `<annotation>` component is concerned, as is the use of `ng-click` and `ng-disabled`.

The purpose of this component is to encapsulate the details of the buttons. In future for example, we might handle labels and tooltips differently depending on the input method (touch vs. mouse). The button will however have a label, respond to input and have some kind of disabled state.